test/test_gitlabtest.yaml

```diff
@@ -1,6 +0,0 @@
-token: gf34bn667fgdg567v3d789g543d4d6f876432456
```

Comment:
if this was your token, please change it asap.

Reply:
This was a fake token that was here before me.